src/seasidecache.cpp

```diff
@@ -1420,6 +1438,7 @@ void SeasideCache::startRequest(bool *idleProcessing)
             m_fetchRequest.setFilter(favoriteFilter());
             m_fetchRequest.setFetchHint(favoriteFetchHint(m_fetchTypes));
             m_fetchRequest.setSorting(m_sortOrder);
+            qDebug() << "Starting favorites query at" << m_timer.elapsed() << "ms";
```

Comment:
You mean, the row counts are reported as -1?  The timings shouldn't be affected?